qiskit/qpy/binary_io/schedules.py

```diff
@@ -88,6 +125,10 @@ def _read_symbolic_pulse(file_obj, version):
         version=version,
         vectors={},
     )
+    if qiskit_version <= (0, 22, 2):
```

Comment:
```suggestion
    if qiskit_version < (0, 23, 0):
```
This PR will not be backported so the minimum version must be 0.23.
https://github.com/Qiskit/qiskit-terra/blob/main/qiskit/VERSION.txt#L1

Reply:
This is intentional because likely compat test installs 0.22.3?
https://dev.azure.com/qiskit-ci/qiskit-terra/_build/results?buildId=41860&view=logs&j=e4ea27c2-3a16-5b32-7be8-0bf30fe9e828&t=b4382375-beb9-5aee-c896-08d1279c743b&l=133

Reply:
Thank you!